README.md

```diff
@@ -1,5 +1,31 @@
-hi yes this is the shitcode of bolb bot. 
+# h
+
+hi yes this is the shitcode of bolb bot.
 have fun.
 
-rename `config.py.example` to `config.py` btw and fill it in.
-written by [koala#9712](https://github.com/koala9712) and [Sham#0002](https://github.com/realShamlol) ofc
\ No newline at end of file
+written by [koala#9712](https://github.com/koala9712), [Sham#0002](https://github.com/realShamlol) and [ooliver1](<https://github.com/ooliver> 1) ofc
```

Comment:
Can you change this to your Discord username#tag, as last time @koala9712 requested I do that, and also please fix your hyperlink as it says `... ooliver> 1` instead of `... ooliver1>`

Reply:
🤔